### PR TITLE
Disallow all robots

### PR DESCRIPTION
**Pull request by brenetic**
We do not want the Self Service app to be indexed.

https://trello.com/c/y02L98Wo/643-add-robotstxt

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template ">
   <head>
+    <meta name=”robots” content=”noindex,nofollow”>
```
Comment (jakubmiarka) [inline app/views/layouts/application.html.erb]:
```suggestion
    <meta name="robots" content="noindex,nofollow">
```
🙈

**Comment by jakubmiarka** [comment]
Rails will not serve the `robots.txt` file unless the `RAILS_SERVE_STATIC_FILES` is set to true. It can be done either in the Dockerfile or terraform fargate task definition.

**Comment by brenetic** [comment]
@jakubmiarka https://github.com/alphagov/verify-infrastructure/pull/287

**Comment by jakubmiarka** [comment]
> @jakubmiarka [alphagov/verify-infrastructure#287](https://github.com/alphagov/verify-infrastructure/pull/287)

Was hoping you would pick up the one-liner Docker option 😄 but this will do too

**Pull request merged**